contrib/scripts/orphan-clean-up/orphan_cleanup.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
```

Comment:
Think it'd be nice to throw in a `set -euo pipefail` here at the top. In case the `curl` fails etc.